packages/story-editor/src/components/floatingMenu/elements/flipHorizontal.js

```diff
@@ -24,13 +24,16 @@ import { __ } from '@googleforcreators/i18n';
  * Internal dependencies
  */
 import { IconButton } from './shared';
+import useFlip from './shared/useFlip';
 
 function FlipHorizontal() {
+  const { horizontal, toggleHorizontal } = useFlip();
```

Comment:
I really like how simple this component is. That's the goal for all of these menu elements!